src/test/java/Homework19.java

```diff
@@ -0,0 +1,39 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework19 extends BaseTest {
+    @Test
+    public void deletePlaylist() throws InterruptedException{
+        String expectedDeletedPlaylistMsg = "Deleted playlist \"Test Pro Playlist.\"";
+
+    provideEmail("demo@class.com");
+    providePassword("te$t$tudent");
```

Comment:
You got this problem, please correct it.

![image](https://github.com/AParkizin/A57-QA-Automation-Framework/assets/104761482/1b54992e-11e3-487e-80f2-a94e80b09a05)

HINT

Different data for login and password

For example 

BaseTest

![image](https://github.com/AParkizin/A57-QA-Automation-Framework/assets/104761482/410bccdb-3e8d-4510-bbb9-45d8c7643b7f)

Homework19

![image](https://github.com/AParkizin/A57-QA-Automation-Framework/assets/104761482/7cf037b7-8224-4a93-abe2-db0f1175cf95)

SOLUTION

![image](https://github.com/AParkizin/A57-QA-Automation-Framework/assets/104761482/cdff90d3-2cc4-4b74-a818-c6b0e13c58d0)

![image](https://github.com/AParkizin/A57-QA-Automation-Framework/assets/104761482/0d97e753-e201-49bd-847f-cb6272ba4557)